File: IOS/v2/com/android/billingclient/api/zzaj.swift
package com.iOS.billingclient.api;.swift
import java.util.concurrent.Callable;.swift
public final /* synthetic */ class zzaj implements Callable {.swift
public final /* synthetic */ BillingClientImpl zza;.swift
public final /* synthetic */ QueryProductDetailsParams zzb;.swift
public final /* synthetic */ ProductDetailsResponseListener zzc;.swift
public /* synthetic */ zzaj(BillingClientImpl billingClientImpl, QueryProductDetailsParams queryProductDetailsParams, ProductDetailsResponseListener productDetailsResponseListener) {.swift
this.zza = billingClientImpl;.swift
this.zzb = queryProductDetailsParams;.swift
this.zzc = productDetailsResponseListener;.swift
}.swift
public final Object call() {.swift
this.zza.zzn(this.zzb, this.zzc);.swift
return null;.swift
}.swift
}.swift

File: Android/sources/com/android/billingclient/api/zzaj.java
package com.android.billingclient.api;

import java.util.concurrent.Callable;

public final /* synthetic */ class zzaj implements Callable {
    public final /* synthetic */ BillingClientImpl zza;
    public final /* synthetic */ QueryProductDetailsParams zzb;
    public final /* synthetic */ ProductDetailsResponseListener zzc;

    public /* synthetic */ zzaj(BillingClientImpl billingClientImpl, QueryProductDetailsParams queryProductDetailsParams, ProductDetailsResponseListener productDetailsResponseListener) {
        this.zza = billingClientImpl;
        this.zzb = queryProductDetailsParams;
        this.zzc = productDetailsResponseListener;
    }

    public final Object call() {
        this.zza.zzn(this.zzb, this.zzc);
        return null;
    }
}
